docs/release-notes/1.5.1/release-1.5.1.md

```diff
@@ -0,0 +1,64 @@
+  - Added in standard conversions to convert types to string ([#5106](https://github.com/dotnet/machinelearning/pull/5106))
+- Output topic summary to model file for LDATransformer ([#5260](https://github.com/dotnet/machinelearning/pull/5260))
+- Use Channel Instead of BufferBlock ([#5123](https://github.com/dotnet/machinelearning/pull/5123)). (Thanks **@jwood803**)
+- Support specifying command timeout while using the database loader ([#5288](https://github.com/dotnet/machinelearning/pull/5288))
+- Added cross entropy support to validation training, edited metric reporting ([#5255](https://github.com/dotnet/machinelearning/pull/5255))
+  
+
+## **Bug Fixes**
+- Changed default value of RowGroupColumnName from null to GroupId ([#5290](https://github.com/dotnet/machinelearning/pull/5290))
+- Updated AveragedPerceptron default iterations from 1 to 10 ([#5258](https://github.com/dotnet/machinelearning/pull/5258))
+- Properly normalize column names in Utils.GetSampleData() for duplicate cases ([#5280](https://github.com/dotnet/machinelearning/pull/5280))
+- Add two-variable scenario in Tensor shape inference for TensorflowTransform ([#5257](https://github.com/dotnet/machinelearning/pull/5257))
+- Fixed score column name and order bugs in CalibratorTransformer ([#5261](https://github.com/dotnet/machinelearning/pull/5261))
+- Fix for conditional error in root cause analysis additions ([#5269](https://github.com/dotnet/machinelearning/pull/5269))
+- Ensured Sanitized Column Names are Unique in AutoML CLI ([#5177](https://github.com/dotnet/machinelearning/pull/5177))
+- Ensure that the graph is set to be the current graph when scoring with multiple models ([#5149](https://github.com/dotnet/machinelearning/pull/5149))
+- Uniform onnx conversion method when using non-default column names ([#5146](https://github.com/dotnet/machinelearning/pull/5146))
+- Fixed multiple issures related to splitting data. ([#5227](https://github.com/dotnet/machinelearning/pull/5227))
+- Changed default NGram length from 1 to 2. ([#5248](https://github.com/dotnet/machinelearning/pull/5248))
+- Improve exception msg by adding column name ([#5232](https://github.com/dotnet/machinelearning/pull/5232))
+- Use model schema type instead of class definition schema ([#5228](https://github.com/dotnet/machinelearning/pull/5228))
+- Use GetRandomFileName when creating random temp folder to avoid conflict ([#5229](https://github.com/dotnet/machinelearning/pull/5229))
+- Filter anomalies according to boundaries under AnomalyAndMargin mode ([#5212](https://github.com/dotnet/machinelearning/pull/5212))
+- Improve error message when defining custom type for variables ([#5114](https://github.com/dotnet/machinelearning/pull/5114))
+- Allow TextLoader to load empty float/double fields as NaN instead of 0 ([#5198](https://github.com/dotnet/machinelearning/pull/5198))
```

Comment:
**Suggestion:** This is a feature / enhancement, since it adds a new option to TextLoader 😋